README.md

```diff
@@ -108,3 +108,12 @@ app.namespace('/v1').get('/', (req, res) => {
   return prism.process({ baseUrl: 'http://x.com/v1', path: '/' });
 });
 ```
+
+## Development
+
+### Common issues
+
+1. `jest --watch` throws ENOSPC error
+
+- [optional] Install `watchman` as per [documentation](https://facebook.github.io/watchman/docs/install.html#installing-from-source)
```

Comment:
@chris-miaskowski in order to run jest in watch mode I need to install watchman? If so, maybe we can add this to the yarn setup command? Our users will be happier.

Reply:
Nope, watchman won't help solving the issue it will only make the error read clearer. The actual error is related to linux OS strictly and the solution is to modify `fs.inotify.max_user_watches`.

Sometimes, e.g. when you have multiple VS Code instances open and those instances watch hundreds of files, jest --watch will throw an error because a maximum of watchers per OS is reached.